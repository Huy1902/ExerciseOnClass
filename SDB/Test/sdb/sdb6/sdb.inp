50
15
8
17
16
37
9
17
20
30
40
17
22
34
4
22
26
7
26
44
41
42
35
17
2
18
46
23
47
32
19
14
42
27
39
6
15
20
45
38
4
17
26
18
4
1
20
20
31
12
39
