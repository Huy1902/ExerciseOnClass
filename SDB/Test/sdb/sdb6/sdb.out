25
8
16
37
9
30
40
34
7
44
41
35
2
46
23
47
32
19
14
27
6
45
38
1
31
12
